src/components/NavBar.jsx

```diff
@@ -240,6 +205,7 @@ export default ({ onExtensionDialogOpen, onCreateAccountDialogOpen }) => {
 									{colorMode === "light"
 										? "Switch to dark mode"
 										: "Switch to light mode"}
+>>>>>>> develop
```

Comment:
Please ensure that all git tags are removed @suyanksaraswat

Reply:
removed.